internal/app/concurrent_proxy_stage_test.go

```diff
@@ -144,42 +147,28 @@ func (s *ConcurrentProxyStage) the_concurrent_requests_are_sent() {
 		return nil
 	})
 
-	if err != nil {
-		s.t.Error(err)
-	}
+	s.require.NoError(err)
```

Comment:
There was a general comment about calling `FailNow` from @henrique-pinto-form3 . Doesn't this method still potentially end up calling `t.FailNow` under the hood?

Reply:
I don't think this is a problem, we're not creating a goroutine as part of the stage setup?

Reply:
I intended to change all to assert, mainly for consistence, just overlooked this file.  Will change.